src/main/java/com/crosswordservice/controller/CrosswordController.java

```diff
@@ -0,0 +1,139 @@
+package com.crosswordservice.controller;
+
+import com.crosswordservice.baseClasses.Configuration;
+import com.crosswordservice.baseClasses.Question;
+import com.crosswordservice.crosswordchecker.CrosswordChecker;
+import com.crosswordservice.repositories.ConfigurationRepository;
+import com.crosswordservice.repositories.QuestionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Rest Controller for the crossword-puzzle backend
+ */
+@RestController
+public class CrosswordController {
+
+    @Autowired
+    QuestionRepository questionRepository;
+    @Autowired
+    ConfigurationRepository configurationRepository;
+
+    @PostMapping("/inputTestData")
+    public List<Question> inputTestData(){
+        Configuration config = new Configuration("test");
+        configurationRepository.save(config);
+
+        List<Question> questions = new ArrayList<>();
+
+        Question quest1 = new Question(config.getId(),"Which language extends Javascript with type safety?","Typescript");
+        Question quest2 = new Question(config.getId(),"How is the system of rules called which defines well-formed expressions?","Syntax");
+        Question quest3 = new Question(config.getId(),"Which loop allows to set the count of iterations in the head?","for-loop");
+        Question quest4 = new Question(config.getId(),"What is the abbreviation of the computing unit in a computer?","CPU");
+        questions.add(quest1);
+        questions.add(quest2);
+        questions.add(quest3);
+        questions.add(quest4);
+        questionRepository.saveAll(questions);
+
+        return questions;
+    }
+
+    /**
+     * Adds a configuration
+     * @param configuration configuration which will be added
+     * @return added configuration
+     */
+    @PostMapping("/configurations")
+    public Configuration saveConfiguration(@RequestBody Configuration configuration){
+        configurationRepository.save(configuration);
+        return configuration;
+    }
+
+    /**
+     * Adds questions to a configuration.
+     * @param questions question which will be added to the configuration
+     * @param name configuration name
+     * @return added questions
+     */
+    @PostMapping("/questions/{name}")
+    public List<Question> saveAllQuestions(@RequestBody List<Question> questions, @PathVariable String name) {
+        Configuration config = configurationRepository.findByName(name);
```

Comment:
I think we should add some Error-Handling if the requested configuration is not present. Maybe its ok for a prototype to ignore this but then we have to do it later.